core/src/main/java/org/opensearch/sql/expression/DSL.java

```diff
@@ -290,10 +290,6 @@ public static FunctionExpression multiply(Expression... expressions) {
     return compile(FunctionProperties.None, BuiltinFunctionName.MULTIPLY, expressions);
   }
 
-  public static FunctionExpression adddate(Expression... expressions) {
```

Comment:
What was the reason for getting rid of all of these functions? Were they simply not used?

Reply:
1. They were used in tests only
2. I rewrote tests and made new DSL definitions for those functions in `DateTimeTestBase`
3. These functions require a `FunctionProperties` object as of now, they crash on `FunctionProperties.None`